README.md

```diff
@@ -1,39 +1,33 @@
-Cassandra Plugins for Apache Pekko Persistence
-======================================
+# Cassandra Plugins for Apache Pekko Persistence
 
 Replicated Apache Pekko Persistence journal and snapshot store backed by [Apache Cassandra](https://cassandra.apache.org/).
 
 Apache Pekko Persistence Cassandra is a fork of [Akka Persistence Cassandra](https://github.com/akka/akka-persistence-cassandra)
 
-Documentation
--------------
+## Documentation
 
-**TODO add documentation links**
+See https://pekko.apache.org for the documentation including the API docs. The docs for all the Apache Pekko modules can be found there.
```

Comment:
Does there have a dedicated page?

Reply:
changed to provide a separate link for this module's docs